subgraphs/curve/subgraph.template.mustache

```diff
@@ -213,7 +213,7 @@ dataSources:
           file: ./abis/Erc20BytesSymbol.json
       eventHandlers:
         # - event: BasePoolAdded(address)
-        #  handler: handleBasePoolAdded
+        #   handler: handleBasePoolAdded
```

Comment:
Is this expected to be commented out?

Reply:
Yeah, for now it is, as this just registers an already created pool as a base pool in the registry, so we don't need to do anything as that pool should already have been indexed previously